slice/Ice/Router.ice

```diff
@@ -40,8 +40,8 @@ interface Router
     ["nonmutating", "cpp:const"] idempotent Object* getServerProxy();
 
     /// Add new proxy information to the router's routing table.
-    /// @param proxies The proxies to add.
-    /// @return Proxies discarded by the router.
+    /// @param proxies The proxies to add. The proxies sequence must not contain any null proxies.
```

Comment:
What about:
@param proxies The proxies to add. Adding a null proxy is an error.
@return Proxies discarded by the router. These proxies are all non-null.